readme.md

```diff
@@ -0,0 +1,105 @@
+# Intro
+
+This is MVP for one of our client (UTSEUS) which helps teams on hackathons exchange skills among team members and pay for their help.  It has three modules: web client (React), server (NodeJS) and storage with some business logic on blockchain (ethereum). This doc descriebs how to setup environment and run server backed by blockchian. 
+
+## Install environment
+
+A whole environment is described in scripts of package.json. Almost all what you need is run this scripts from the server folder. 
+
+```
+npm run install
+npm run testrpc
```

Comment:
This line starts a server so can't proceed to the next line. Should explain this separately.

Reply:
Done